README.md

```diff
@@ -85,6 +85,8 @@ A curated list of resources for learning and programming in Noir.
 
 ### Libraries
 
+[Noir Directory](https://noir.directory) - All Noir libraries in a single place. The npm of Noir.
```

Comment:
```suggestion
[Noir Directory](https://noir.directory) - A directory of Noir libraries.
```
Suggesting to not call it the `npm` of Noir since it's a bit simpler